src/main/resources/templates/tutorAssignment/manage.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" lang="en">
+
+<head>
+  <div th:replace="bootstrap/title.html"></div>
+  <div th:replace="bootstrap/bootstrap_head.html"></div>
+</head>
+
+<body>
+  <div class="container">
+    <div th:replace="bootstrap/bootstrap_nav_header.html"></div>
+
+    <div class="row">
+      <div class="col-md-6">
+        <h2 th:text="${courseOffering.quarter + ' ' + courseOffering.courseId}"></h2>
+        <table class="table table-striped table-responsive-md">
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Assigned?</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr th:each="tutor : ${tutors}">
+              <td th:text="${tutor.firstName + ' ' + tutor.lastName}"></td>
+              <td>
+                <input class="tutorAssignmentCheckBox" type="checkbox" th:checked="${shouldBeChecked.test(tutor)}"
+                  th:attr="data-tid=${tutor.id},data-cid=${courseOffering.id}" />
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+
+    <div th:replace="bootstrap/bootstrap_footer.html"></div>
+  </div>
+  <div th:replace="bootstrap/bootstrap_scripts.html"></div>
+  <script th:inline="javascript">
+    /*<![CDATA[*/
+    $(".tutorAssignmentCheckBox").click(e => {
+      e.preventDefault()
+      e.stopImmediatePropagation()
+
+      if (e.target.checked) {
+        $.ajax({
+          url: "/tutorAssignment/add",
+          method: "POST",
+          data: {
+            tid: e.target.dataset.tid,
+            cid: e.target.dataset.cid,
+            /*[[${_csrf.parameterName}]]: [[${_csrf.token}]]*/
```

Comment:
Why is this being left behind?

Reply:
All of those comments are specific to the thymeleaf preprocessor. See https://www.thymeleaf.org/doc/tutorials/2.1/usingthymeleaf.html#script-inlining-javascript-and-dart for details